pkg/manager/member/pd_scaler_test.go

```diff
@@ -577,3 +577,11 @@ func normalPDMember(tc *v1alpha1.TidbCluster) {
 		ordinalPodName(v1alpha1.PDMemberType, tcName, 4): {Health: true},
 	}
 }
+
+func newPVCWithDeleteAnnotaion(set *apps.StatefulSet, memberType v1alpha1.MemberType, name string, ordinal int32) *corev1.PersistentVolumeClaim {
+	pvc := _newPVCForStatefulSet(set, memberType, name, ordinal)
+	pvc.Annotations = map[string]string{
```

Comment:
how about checking if any `Annotations` already set by (a feature version of) `_newPVCForStatefulSet`?